neps/nep-0148.md

```diff
@@ -35,34 +36,33 @@ A fungible token smart contract allows for discoverable properties. Some propert
 #### Token provides metadata upon deploy and initialization
 
 Alice deploys a wBTC fungible token contract.
-
-**Assumptions**
+Assumptions
 
 - The wBTC token contract is `wbtc`.
 - Alice's account is `alice`.
 - The precision ("decimals" in this metadata standard) on wBTC contract is `10^8`.
 
-**High-level explanation**
+##### High-level explanation
 
 Alice issues a transaction to deploy and initialize the fungible token contract, providing arguments to the initialization function that set metadata fields.
 
-**Technical calls**
+##### Technical calls
 
 1. `alice` deploys a contract and calls `wbtc::new` with all metadata. If this deploy and initialization were done using [NEAR CLI](https://docs.near.org/tools/near-cli) the command would be:
 
-    near deploy wbtc --wasmFile res/ft.wasm --initFunction new --initArgs '{
```

Comment:
Let's put it into fences as otherwise the indentation is ruined

Reply:
In fact the linter suggests to use fences. However I disabled that check to reduce amount of modifications. However, it seams that auto-fix modified it anyway.